src/Tribe/Views/V2/Widgets/Widget_Abstract.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+namespace Tribe\Events\Views\V2\Widgets;
+
+use Tribe\Events\Views\V2\View;
+use Tribe\Events\Views\V2\View_Interface;
+use Tribe__Context as Context;
+
+/**
+ * The abstract all widgets should implement.
+ *
+ * @since   TBD
+ *
+ * @package Tribe\Widget
+ *
+ */
+abstract class Widget_Abstract extends \Tribe\Widget\Widget_Abstract {
```

Comment:
:no_entry_sign: Additional blank lines found at end of doc comment

<sub>hash: ce7bb5c2e5c1fd7bfd21cace06ea4885</sub>